### ISSUE TITLE
Recall / Accuracy Questions on Paper

### ISSUE THREAD

**Issue by okay-okay** [facebookresearch/AVT#21]
Hi Rohit, just had a few questions when re-reading the paper:

1. In Table 4, it seems that accuracy on the new proposed model is worse, but recall is better (compared to other models), do you have some insight into this tradeoff? Your loss metric is optimizing for accuracy, so I'm just curious if there's anything else you might be doing.

2. Do you have any results on AVT-h and AVT-b without fine-tuning on AVT-b? If not, do you have any idea on how it performs without fine-tuning?

3. Why does replacing the backbone with AVT-b significantly boost recall, but not top-1 accuracy?

**Comment by rohitgirdhar** [comment]
Hi,
1. Yes, we observed relatively lower performance on top-1/5 metrics. We generally found AVT model to perform better on tail classes (also observed that on EK100), and since the recall metric equally weighs each class we end up performing better there. The Im21K pretraining used in the AVT-b backbone might also help in recognizing tail classes.
2. No we don't; we always finetuned it end-to-end. Though with the released code it should be possible to do so.
3. Answered in 1.

Hope that helps!
